sidhariyal/reference.fab

```diff
@@ -0,0 +1,42 @@
+# Reference file for Western skyculture

```

Comment:
This should list references for the sidhariyal skyculture.

Reply:
Noted with thanks, I'll incorporate it.